example/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
 
         <activity
             android:name=".activity.PaymentActivity"
-            android:theme="@style/Theme"
+            android:theme="@style/Theme.AppCompat"
```

Comment:
The theme of the activity had to be changed because I switched it from a FragmentActivity to an AppCompatActivity. This is a good thing.